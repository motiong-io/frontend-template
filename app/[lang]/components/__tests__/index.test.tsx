import { render } from "@testing-library/react";
import LocaleSwitcher from "../locale-switcher";

describe("LocaleSwitcher", () => {
  it("render result match snapshot", () => {
    const { baseElement } = render(<LocaleSwitcher />);
    expect(baseElement).toMatchSnapshot();
  });
});
