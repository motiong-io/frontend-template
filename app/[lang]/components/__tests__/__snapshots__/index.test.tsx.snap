// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocaleSwitcher render result match snapshot 1`] = `
<body>
  <div>
    <div
      class="switcher"
    >
      <div
        class="item"
      >
        <a
          href="/"
        >
          中文
        </a>
      </div>
      <div
        class="item"
      >
        <a
          href="/"
        >
          English
        </a>
      </div>
    </div>
  </div>
</body>
`;
